apps/roam/src/components/DiscourseContext.tsx

```diff
@@ -344,6 +345,9 @@ export const ContextContent = ({ uid, results }: Props) => {
             },
           },
         }));
+        posthog.capture("Discourse context: Show result", {
```

Comment:
Let's move this to the when the caret is clicked to open.  But only on open, not close.


![discourse-context](https://github.com/user-attachments/assets/208f27d9-a487-4a60-9132-6cb870a49db8)

https://github.com/DiscourseGraph/discourse-graph/blob/c21cc0bdd04813bedeb809177f99f2c06c2b8f45/apps/roam/src/components/DiscourseContext.tsx#L439

Reply:
nit: Also rename to "Discourse Context: Show Results" (Pascal Case)